x-pack/plugins/monitoring/public/directives/cluster/listing/index.js

```diff
@@ -342,6 +353,10 @@ const handleClickInvalidLicense = (scope, clusterName) => {
   });
 };
 
+const formatClusters = clusters => {
```

Comment:
Function was probably doing something, but it's a noop currently.

Reply:
Removed!